File: C++/روز آزادی بیان در برره.cpp
#include <iostream>
using namespace std;
int main()
{
    int n,j=0,s;
    cin>>n;
    s=n;
    for(int i=1;i<=n;i++)
    {
        while(s>=0)
        {
            s-=i;
            j++;
        }
    }
    if(j%2==0)
       cout<<"Payin Barare";
    else
        cout<<"Bala Barare";
}

File: Python/روز آزادی بیان در برره.py
count_bala = 1
count_pain = 2
k = int(input())
while True:
    if count_bala == k:
        print('Payin Barare')
        break
    elif count_pain == k:
        print('Bala Barare')
        break
    count_bala += 2
    count_pain += 2
